### PR TITLE
feat(qa): improve issue templating (new GitHub issues templates)

### PR DESCRIPTION
**Pull request by SuperITMan**
ISSUES CLOSED: #37

## PR Checklist

Please check if your PR fulfills the following requirements:

-   [X] The commit message follows our guidelines: https://github.com/NationalBankBelgium/ngx-form-errors/blob/master/CONTRIBUTING.md#-commit-message-guidelines
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #37 

## What is the new behavior?

A template per each type of issue we could open:
  1. Bug report
  2. Feature request
  3. Documentation
  4. Support
  5. Other

## Does this PR introduce a breaking change?

```
[ ] Yes
[X] No
```

## Other information

To realize this PR, I took inspiration on:
  - https://github.com/facebookexperimental/rome
  - https://github.com/angular/angular

You can see the result on https://github.com/SuperITMan/ngx-form-errors/issues/new/choose

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/02-feature_request.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## What is the motivation / use case for changing the behavior?
+
+<!-- Describe the motivation or the concrete use case. -->
+
+## Environment
+
+<pre><code>
+Angular version: X.Y.Z
+NgxFormErrors version: X.Y.Z
```
Comment (christophercr) [inline .github/ISSUE_TEMPLATE/02-feature_request.md]: I'm not sure if these 2 options are applicable for a "feature request". Or at least the "NgxFormErrors version" which would be all the previous versions actually...

File: .github/ISSUE_TEMPLATE/05-other.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+name: '💭 Other'
+about: Issues that don't fit under anything else
+title: ''
+labels: ''
```
Comment (christophercr) [inline .github/ISSUE_TEMPLATE/05-other.md]: Maybe we could add also these generic sections to guide the developer and to make it more structured for everyone:

```
What:

Why:

How:
```

File: .github/ISSUE_TEMPLATE/02-feature_request.md (hunk 1 above)
Comment (SuperITMan) [reply .github/ISSUE_TEMPLATE/02-feature_request.md]: You're right 😛 
So should I remove all the environment stuff or does it make sense to keep it according to you ?

**Comment by SuperITMan** [comment]
@christophercr I updated my PR. I decided to remove all the information about the environment for the feature request 😊 
Because I think it's not relevant. 
Could you please have a look?

**Pull request merged**